lib/node_modules/@stdlib/array/base/broadcasted-quinary4d/README.md

```diff
@@ -0,0 +1,159 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# bquinary4d
+
+> Apply a quinary callback to elements in four [broadcasted][@stdlib/array/base/broadcast-array] nested input arrays and assign results to elements in a four-dimensional nested output array.
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var bquinary4d = require( '@stdlib/array/base/broadcasted-quinary4d' );
+```
+
+#### bquinary4d( arrays, shapes, fcn )
+
+Applies a quinary callback to elements in four [broadcasted][@stdlib/array/base/broadcast-array] nested input arrays and assigns results to elements in a four-dimensional nested output array.
+
+```javascript
+var zeros4d = require( '@stdlib/array/base/zeros4d' );
+
+function add( x, y, z, w, v ) {
+    return x + y + z + w + v;
+}
+
+var x = [ [ [ [ 1.0, 2.0 ] ] ] ];
+var y = [ [ [ [ 3.0 ], [ 4.0 ] ] ] ];
+var z = [ [ [ [ 5.0 ] ] ] ];
+var w = [ [ [ [ 2.0 ] ] ] ];
+var v = [ [ [ [ 1.0 ] ] ] ];
+var out = zeros4d( [ 2, 2, 2, 2 ] );
+
+var shapes = [
+    [ 1, 1, 1, 2 ],
+    [ 1, 1, 2, 1 ],
+    [ 1, 1, 2, 2 ],
+    [ 1, 2, 1, 1 ],
+    [ 2, 2, 2, 2 ],
```

Comment:
@Neerajpathak07 These shapes are incorrect. For `z`, `w`, and `v` the shape is `[ 1, 1, 1, 1 ]`.